src/java/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ endif()
 include(UseSWIG)
 include(UseJava)
 
-set(CMAKE_SWIG_OUTDIR ${CMAKE_CURRENT_SOURCE_DIR}/io/openthread/commissioner)
+set(CMAKE_SWIG_OUTDIR ${CMAKE_CURRENT_BINARY_DIR}/io/openthread/commissioner)
```

Comment:
We cannot directly use `CMAKE_SWIG_OUTDIR` because C++ wapper files are also outputted there.